draft-ietf-scone-protocol.md

```diff
@@ -550,8 +659,51 @@ discards a SCONE packet without also successfully processing another packet
 from the same datagram SHOULD ignore any throughput advice signal. Such a datagram
 might be entirely spoofed.
 
+Once negotiated, endpoints that seek to receive throughput advice on a flow
+MUST send a SCONE packet at least twice each monitoring period; see {{time}}.
+
+Sending SCONE packets more often might be necessary to:
+
+Avoid missing advice:
+: If SCONE packets are not sent, updated, and received
+  for an entire monitoring period,
+  an application might incorrectly assume that no advice is being provided.
+
+Reduce latency:
+: The time between SCONE packets determines the maximum delay
+  between changes in throughput advice
+  and when that advice can be received and acted upon.
+
+A sender can track the receipt of the coalesced QUIC packet
+and send another SCONE packet if when loss is detected.
+However, it is likely simpler to send SCONE packets more often.
+
+Sending a SCONE packet every 20&ndash;30 seconds
+is likely sufficient to ensure that throughput advice is not lost,
+though endpoints might send a packet every few seconds
+to improve responsiveness.
+This period could be determined by how quickly an application
+is able to respond to a change in throughput advice.
+
+For example, a streaming application
+that fetches video segments that are 5 seconds in length
+might send SCONE packets on a similar cadence.
+A real-time conferencing application might send more often.
+In either case, the length of the monitoring period ({{time}})
+limits how fast any application can react.
+
+Though sending SCONE packets more than once each round trip time
+might help reduce exposure to packet loss,
+it is better to spread updates over time
+rather than to send multiple SCONE packets in less frequent bursts.
+
+The main cost associated with sending SCONE packets
```

Comment:
Is there any compute cost on the network element that we should mention?

Reply:
I am going to assume that the cost is modest because the network element can (now) ignore most SCONE packets.  Any they do update will require effort, but that's effort that they balance against the benefits they deliver (better user experience, better adherence to network policy, etc...).